temporalcli/devserver/server.go

```diff
@@ -321,13 +321,7 @@ func (s *StartOptions) buildSQLConfig() (*config.SQL, error) {
 		conf.DatabaseName = s.DatabaseFile
 	}
 	for k, v := range s.SqlitePragmas {
-		// Only some pragmas allowed
-		switch k {
-		case "journal_mode", "synchronous":
-		default:
-			return nil, fmt.Errorf("unrecognized pragma %q, only 'journal_mode' and 'synchronous' allowed", k)
-		}
-		conf.ConnectAttributes["_"+k] = v
+		conf.ConnectAttributes[k] = v
```

Comment:
Interesting, I suspect we just didn't have test cases. Would you be willing to add a test (or adjust an existing one) that just adds any innocuous pragma just to ensure it works?

Reply:
Sure, I'll add one